smoke-tests/src/test/java/org/hypertrace/agent/smoketest/SpringBootSmokeTest.java

```diff
@@ -113,6 +113,10 @@ public void get() throws IOException {
                 .map(attribute -> attribute.getValue().getStringValue())
                 .count()
             > 0);
+    // TODO test the container ID once we switch from Zipkin exporter
+    // Zipkin exporter does not add resource attributes to span
+    // https://github.com/open-telemetry/opentelemetry-java/issues/1970
```

Comment:
cc) @davexroth https://github.com/open-telemetry/opentelemetry-java/issues/1970

Reply:
hmm, so what do we want to do in the mean time, as the platform needs the container id?  Should we just add it as an attribute for now?